### PR TITLE
MLH Prep logo substituted + Footer added

### PR DESCRIPTION
**Pull request by sumitshinde2608**
## Please, go through these steps before you submit a PR.

- [ ] My Pod Leader knows I'm working on this Pull Request
- [ ] I've explained what the Pull Request is adding.
- [ ] I've explained why this is important.

- There was nothing to be done much, just an image source change and a sass change which completes the assigned issue at hand.

### CODE REVIEW

**Comment by khattakdev** [comment]
@sumitshinde2608 Please tag someone for review

**Comment by sumitshinde2608** [comment]
@GustavoElizarraras @P0intMaN If you guys could review the code changes :)

**Review by GustavoElizarraras — COMMENTED** [review]
The changes in _Gemfile.lock_ are specific of your OS/machine, aren't they? So, if it get merged, everyone enviroment could stop working, isn't it?
Regarding The HTML and CSS, looks fine to me.

**Comment by sumitshinde2608** [comment]
@GustavoElizarraras as far as I remember we have not touched the _Gemfile_ so whatever changes that have occurred in the _Gemfile.lock_ should not affect the working when run on other systems

**Comment by P0intMaN** [comment]
You are right @sumitshinde2608 , seems like bundles are created and adjusted based on the system it is running on. I don't think it would call for a problem.

**Comment by sumitshinde2608** [comment]
Guys if you could review this soon as we have one more PR to make regarding the footer (-_-;).

**Comment by P0intMaN** [comment]
I think the footer is a part of this PR... right? Coz I saw the `footer.html` for it. Anyways, LGTM. Safe to merge.

**Review by P0intMaN — APPROVED** [review]

File: _includes/footer.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+                    <div class="col-lg-4">
+                        <div class="mt-4">
+                            <h5 class="f-18 main-text">Features</h5>
+                            <ul class="list-unstyled footer-link mt-3">
+                                <li><a href="#">Meet the Pod </a></li>
+                                <li><a href="#">Pod stats</a></li>
+                                <li><a href="#">Leaderboard</a></li>
+                                <li><a href="#">Projects</a></li>
+                            </ul>
```
[the remark this answers is not in the document]
Comment (sumitshinde2608) [reply _includes/footer.html:29]: Ya, we'll do them in another PR. Actually, we should have started working on the footer issue by creating a branch but it was too late before we caught our mistake. Once this PR gets closed, we'll update our repos and work on the footer in a separate PR

[the remark this answers is not in the document]
Comment (P0intMaN) [reply _includes/footer.html:29]: sounds awesome

**Review by GustavoElizarraras — APPROVED** [review]
Thank you for the clarification @sumitshinde2608  @P0intMaN

**Comment by sumitshinde2608** [comment]
@khattakdev can you please approve these changes so that we can start working on the other PR

**Review by P0intMaN — APPROVED** [review]
Approving again, coz its still asking for a pending review

**Comment by P0intMaN** [comment]
Oh, I got it. @sumitshinde2608, you would have to mark the conversation as "resolved", then it would get approved

**Review by P0intMaN — APPROVED** [review]
GTG!

**Comment by khattakdev** [comment]
@sumitshinde2608 Can you please resolve the merge conflicts so I can merge these changes?

**Comment by sumitshinde2608** [comment]
@GustavoElizarraras @P0intMaN I have added the footer code changes too in the latest commit which solves both my assigned issue. Please review the changes once so that it is cool to merge.

**Comment by sumitshinde2608** [comment]
@khattakdev I have solved the merge conflict. Once the changes get approved, the commit is good to be merged

**Review by P0intMaN — APPROVED** [review]
great! green flag from me!

**Review by GustavoElizarraras — APPROVED** [review]
Good to go @sumitshinde2608

**Review by khattakdev — APPROVED** [review]

**Pull request merged**